lib/dialects/postgres/index.js

```diff
@@ -49,6 +49,7 @@ class PostgresDialect extends AbstractDialect {
     TSVECTOR: true,
     deferrableConstraints: true,
     searchPath: true,
+    notValid: true,
```

Comment:
this needs to be declared in abstract/index.ts, along with a default value.

I think it would be better to group this flag with the other constraint flags (`constraints.notValid`), along with a small comment about its purpose (e.g. supports the "NOT VALID" option on constraints)